src/sentry/tasks/integrations/github/open_pr_comment.py

```diff
@@ -74,3 +76,22 @@ def get_pr_filenames(
     # new files will not have sentry issues associated with them
     pr_filenames: List[str] = [file["filename"] for file in pr_files if file["status"] != "added"]
     return pr_filenames
+
+
+def get_projects_and_filenames_from_source_file(
+    pr_filename: str,
+) -> Tuple[Set[Project], Set[str]]:
+    query = (
```

Comment:
🧐 

this seems odd... we're passing a raw sql query??
Is there any better way to query this model??

Reply:
i couldn't find a "reverse LIKE" in django -- basically matching a substring of an entry in the table to a search string

Reply:
i found a workaround